lib/theme.json

```diff
@@ -246,7 +246,8 @@
 								"margin-block-end": "0"
 							}
 						}
-					]
+					],
+					"marginSelector": " > "
```

Comment:
I guess the benefit of declaring the selector here is having a single source of truth between editor and front end code. If this becomes unnecessary though (say we find a better solution with cascade layers) how easily can we remove it?

Reply:
Yeah, the benefit was to try to keep the selector in one place. It should be easy enough to remove from `theme.json` since it's an undocumented part of the layout definitions object, which isn't intended to be overridden by themes. We'd just need to land the changes to the PHP / JS code that uses a different approach _first_, and then as a final step, remove this property in a follow-up.